### ISSUE TITLE
Are class weights computed in the loss function?

### ISSUE THREAD

**Issue by Ao-Lee** [MeioJane/CHR#2]
thanks for the great work.  I see that the loss of the netwok is a self implemented binary cross entropy loss parameterized by class weights. (I guess the weights are used to adress the imbalanced data labels?). I read the code in main.py, but I didn't see the weights take part in any computation
`-(target*torch.log(input+eps)+(1-target)*torch.log(1-input+eps)) `
Would this be updated later? Thanks

**Comment by MeioJane** [comment]
The code have been updated. The class weights loss is in main.py file.
